File: src/main/java/com/springMvcPractice/application/service/DrillService.java
package com.springMvcPractice.application.service;

import com.springMvcPractice.domain.model.Drill;

import java.util.HashSet;


public interface DrillService {
    HashSet<Drill> createDrillList(final int Cnt, final int drillSize, final int stopSize);

    String convertDrillString(String operandLeft, String operandRight);
}

File: src/main/java/com/springMvcPractice/application/service/DrillService.kt
package com.springMvcPractice.application.service

import com.springMvcPractice.domain.model.Drill

interface DrillService {
    fun createDrillList(cnt: Int, drillSize: Int, stopSize: Int): HashSet<Drill>
    fun convertDrillString(operandLeft: String, operandRight: String): String
}
